includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -85,12 +85,12 @@ private function getLangWithFallback( $lang ) {
 			'user-register' => 'https://www.wikia.com/register',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "user-register" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)